x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -53,6 +53,7 @@ testClusters.configureEach {
   setting 'indices.lifecycle.history_index_enabled', 'false'
   setting 'slm.history_index_enabled', 'false'
   setting 'stack.templates.enabled', 'false'
+  setting 'xpack.ent-search.enabled', 'false'
```

Comment:
ℹ️ Disabling our module since these test cases does not play well with built-in templates (see other config)